roles/user_management/tasks/main.yml

```diff
@@ -29,7 +29,7 @@
   authorized_key:
     user: "{{ item.name }}"
     state: "{{ item.state }}"
-    key: "{{ item.sshkey }}"
+    key: "{{ item.sshkeys }}"
```

Comment:
I'm not sure this is correct as:
1.  documantation claims correct key is [`sshkey`](https://github.com/beskar-cloud/infra-config/tree/main/roles/user_management#role-variables)
2. Also example group vars point out [`sshkey`](https://github.com/beskar-cloud/infra-config/blob/main/group_vars/all.yml#L11-L17)

I would be just fine to change the key to `sshkeys` (changing from scalar string to list of strings), but we need to change it on the other places too.

Reply:
My bad I just now realize that I will close PR